src/clojure/nrepl/core.clj

```diff
@@ -64,6 +64,14 @@
     (concat head (take 1 tail))))
 
 (defn- delimited-transport-seq
+  "Sends a message via client and returns the head of client's response seq
+   by following the steps described below.
+
+    - Add an :id to the message(argument)
```

Comment:
I guess here it'd be good to clarify that the argument in question is the argument of the function that this function returns. A bit meta, right? :-)